test/src/main/java/org/apache/accumulo/test/AuditMessageIT.java

```diff
@@ -207,7 +207,8 @@ public void testTableOperationsAudits() throws AccumuloException, AccumuloSecuri
 
     // Connect as Audit User and do a bunch of stuff.
     // Testing activity begins here
-    auditAccumuloClient = getCluster().getAccumuloClient(AUDIT_USER_1, new PasswordToken(PASSWORD));
+    auditAccumuloClient = getCluster().createAccumuloClient(AUDIT_USER_1,
```

Comment:
why not close this?

Reply:
I wanted to limit the size of this PR.  I won't close #820 until everything has been closed or other issues have been created for remaining work